gum/backend-qnx/gumprocess-qnx.c

```diff
@@ -231,31 +232,55 @@ void
 gum_process_enumerate_modules (GumFoundModuleFunc func,
                                gpointer user_data)
 {
-  GumDlIteratePhdrContext ctx = { func, user_data };
+  gchar * as_path;
+  gint fd, res;
+  gboolean carry_on = TRUE;
+  procfs_mapinfo * mapinfos;
+  gint num_mapinfos;
+  procfs_debuginfo * debuginfo;
+  gint i = 0;
```

Comment:
Regarding ordering of variables, we always order them chronologically based on when they are used in the function. The only exception is the return value, if the function has one, which then always goes first.